src/components/field-types/TextFieldComponent.vue

```diff
@@ -1,25 +1,26 @@
 <template>
-  <validate :state="state" :custom="{'validators': validators}">
+  <validate :state="state" :custom="{'validate': validate(field)}">
     <div class="form-group">
       <label :for="field.id">{{ field.label }}</label>
 
-      <input :id="field.id"
-             v-model="localValue"
-             :type="field.type"
-             :name="field.id"
-             class="form-control form-control-lg"
-             :class="{ 'is-invalid' : state && (state.$touched || state.$submitted) && state.$invalid}"
-             :aria-describedby="field.id + '-description'"
-             :required="field.required"
-             :disabled="field.disabled">
+      <input
+        :id="field.id"
+        v-model="localValue"
+        :type="field.type"
+        :name="field.id"
+        class="form-control form-control-lg"
+        :class="{ 'is-invalid' : state && (state.$touched || state.$submitted) && state.$invalid}"
+        :aria-describedby="field.id + '-description'"
+        :required="field.required"
+        :disabled="field.disabled">
 
       <small :id="field.id + '-description'" class="form-text text-muted">
         {{ field.description }}
       </small>
 
       <field-messages :name="field.id" show="$touched || $submitted" class="form-control-feedback">
         <div slot="required">This field is required</div>
-        <div slot="validators">{{ validationMessage }}</div>
+        <div slot="validate">Validation failed</div>
```

Comment:
maybe keep as injected validationMessage to be able to use it in combination with molgenis i18n